sdcm/scan_operation_thread.py

```diff
@@ -73,11 +78,6 @@ def randomly_add_timeout(cmd) -> str:
     def randomly_form_cql_statement(self) -> Optional[str]:
         ...
 
-    def create_session(self, db_node: BaseNode):
-        credentials = self.db_cluster.get_db_auth()
-        username, password = credentials if credentials else (None, None)
-        return self.db_cluster.cql_connection_patient(db_node, user=username, password=password)
```

Comment:
because the 2 above linse are not actually needed every call, thus they were moved to init. so only a single line left here..

Reply:
so why not like this ?:
```
 def create_session(self, db_node: BaseNode, **connection_kwargs):
        credentials = self.db_cluster.get_db_auth()
        username, password = credentials if credentials else (None, None)
        return self.db_cluster.cql_connection_patient(db_node, user=username, password=password, **connection_kwargs)

# and the call it like this when needed:
with self.create_session(db_node, connect_timeout=300) as session:

```

Reply:
There's no need to repeat these 2 lines:
```
        credentials = self.db_cluster.get_db_auth()
        username, password = credentials if credentials else (None, None)
```
Once it is set in object init time, it's good for the whole run, no need to generate it again.

Reply:
it may, indeed, change during the test... we have a nemesis that is supposed to toggle LDAP configuration on and off, so the credentials shall change, during the test

Reply:
( code was removed )